addons/mail/static/src/xml/activity.xml

```diff
@@ -45,7 +45,7 @@
         <div>
             <textarea class="form-control" rows="3" id="activity_feedback" placeholder="Write Feedback"/>
             <div class="mt8">
-                <button type="button" class="btn btn-xs btn-primary o_activity_popover_done_next">
+                <button type="button" class="btn btn-xs btn-primary o_activity_popover_done_next" t-att-data-activity-type-id="activity_type_id">
```

Comment:
Rename to t-att-data-previous-activity-type-id . Update accordingly remaining code.